core-ui/src/components/Predefined/Create/Deployments/templates.js

```diff
@@ -44,6 +44,7 @@ export function createDeploymentTemplate(namespaceId) {
           },
         },
         spec: {
+          imagePullSecrets: [],
```

Comment:
If it's an array then maybe we should add an option (at least in an advanced form) to add multiple secrets?

Reply:
We talked about it during the daily and Valentin said that's enough for now.